### ISSUE TITLE
Coin -> Asset ID

### ISSUE THREAD

**Issue by adlerjohn** [FuelLabs/block-explorer-v2#35]
Rename coin to asset ID

**Comment by sjblurton** [comment]
just to be clear, all occurrences of coin need changing to assetID? uppercase I, uppercase D?

**Comment by adlerjohn** [comment]
Not necessarily, it depends on context (not sure if the current code is consistent with how "coin" is used). "Coin" and "color" might both be used throughout to mean the same thing, the asset ID. Hence the need to change references to "coin" and "color" to "asset ID."

Uppercase I and D, with a space.
